flink/assets/service_checks.json

```diff
@@ -0,0 +1 @@
+[]
```

Comment:
are there any service checks?

Reply:
I don't think I can include any because we don't ever connect to flink; flink just sends metrics via the datadog API